src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -361,6 +361,7 @@ public static IFullNodeBuilder UseBaseFeature(this IFullNodeBuilder fullNodeBuil
                 .AddFeature<BaseFeature>()
                 .FeatureServices(services =>
                 {
+                    services.AddSingleton(fullNodeBuilder.Network.Consensus.ConsensusFactory);
```

Comment:
Is this really needed?

Reply:
Ah its injected in the deserializer ok yeah fine.

Reply:
That's the purpose of this PR ;)